### DISCUSSION TITLE
Media Library - Cropped images are heavy

### DISCUSSION THREAD

**Discussion by octoduchilli** [strapi/strapi#7954] [Legacy Topics]
Hello,

When images are cropped with strapi, it is normal that they are heavy ?
Strapi blog example: [video](https://www.loom.com/share/10789baf57844ac49f52a3dc305494cf)
In this video, the base image size is 451Kb and the cropped size is 4Mb.

Thanks !

**Comment by derrickmehaffy** [comment]
Closing/locking in favor of https://github.com/strapi/strapi/issues/7990
